pwnlib/util/misc.py

```diff
@@ -216,6 +225,8 @@ def run_in_new_terminal(command, terminal = None, args = None):
         elif 'DISPLAY' in os.environ and which('x-terminal-emulator'):
             terminal = 'x-terminal-emulator'
             args     = ['-e']
+            if isinstance(command, str):
+                command  = shlex.split(command)
```

Comment:
String splitting is a convenience for basic commands.  

`shlex` doesn't follow the same string-splitting as *every* shell, and does not understand most Unix escapes.

If your command is complicated you **MUST** use the `list` of arguments.  Perhaps we should deprecate string splitting altogether?

Reply:
I thought maybe to mimic system and use /bin/bash to split the args,
command = ["/bin/bash", "-c", command].
do you think it is a good idea?

Reply:
You'll want to use `/bin/sh`, but that could work, sure.